ci-operator/config/stackrox/stackrox/README.md

```diff
@@ -1,3 +1,15 @@
 # StackRox OpenShift CI configs
 
-When creating Pull Requests, mind [Making Changes to OpenShift CI](https://docs.engineering.redhat.com/display/StackRox/Making+changes+to+OpenShift+CI) and `Pull Request checklist` therein.
\ No newline at end of file
+https://prow.ci.openshift.org/?repo=stackrox%2Fstackrox
+
+When creating Pull Requests, mind [Making Changes to OpenShift
+CI](https://docs.engineering.redhat.com/display/StackRox/Making+changes+to+OpenShift+CI)
+and `Pull Request checklist` therein.
+
+## Release Template
+
+`stackrox-stackrox-release-x.y*.yaml` files are template YAML used by automation
+to create config for release at branch time. 
+
+*When changes are made to master or nightlies yaml consider if the changes are
+also appropriate for release branches.*
```

Comment:
In addition (or instead of this) I suggest we rely on the `Pull Request Checklist`. I updated the one by adding the item number 3 there. PTAL and edit as you see fit.

Reply:
updated.